### PR TITLE
event category changes

### PR DESCRIPTION
**Pull request by gaurav08suri**

### CODE REVIEW

File: events/views.py [hunk 1]
```diff
@@ -17,7 +17,19 @@ class EventViewSet(ModelViewSet):
     permission_classes = (IsAuthenticatedOrReadOnly,)
     queryset = Event.objects.all()
     serializer_class = EventSerializer
-    filter_fields = ['id', 'name', 'center', 'year', 'event_code', 'gender', 'min_age', 'max_age', 'active']
+    filter_fields = ['id', 'name', 'center', 'year', 'event_code', 'gender', 'min_age', 'max_age', 'active', 'category']
+
+
+
+class EventCategoryViewSet(ModelViewSet):
+    """This endpoint Represents the Centers
```
Comment (roguesherlock) [inline events/views.py]: Centers?

File: events/serializers.py [hunk 2]
```diff
@@ -76,6 +89,7 @@ class Meta:
 
     def create(self, validated_data):
         # event_data = validated_data.pop('event')
+        print(validated_data)
```
Comment (roguesherlock) [inline events/serializers.py]: don't need this ?

File: events/models.py [hunk 3]
```diff
@@ -27,6 +33,7 @@ class Event(models.Model):
     name = models.CharField(max_length=50, help_text=_("Event Name"))
     venue = models.ForeignKey(Address, on_delete=models.CASCADE)
     center = models.ForeignKey(Center, on_delete=models.CASCADE, help_text=_("Center"))
+    category = models.ForeignKey(EventCategory, on_delete=models.CASCADE, help_text=_("Event Category"), default="")
```
Comment (roguesherlock) [inline events/models.py]: should the foreign key have a default of empty string ?

**Comment by gaurav08suri** [comment]
@roguesherlock - Glad to see your review :)

File: events/views.py (hunk 1 above)
Comment (gaurav08suri) [reply events/views.py]: Copy paste issue ... will resolve

File: events/serializers.py (hunk 2 above)
Comment (gaurav08suri) [reply events/serializers.py]: Sure...will remove...was added for debugging purpose

File: events/models.py (hunk 3 above)
Comment (gaurav08suri) [reply events/models.py]: Shouldn't have, but this is creating issues for existing records...so added this...do you know of a better way to handle this ?

File: events/serializers.py (hunk 2 above)
Comment (gaurav08suri) [reply events/serializers.py]: Done

File: events/views.py (hunk 1 above)
Comment (gaurav08suri) [reply events/views.py]: Done

File: events/models.py (hunk 3 above)
Comment (roguesherlock) [reply events/models.py]: A better way would be the set the default to null for existing entries ? Anyways, it's fine for now :)

**Pull request merged**